scripts/BUILD.bazel

```diff
@@ -147,3 +148,18 @@ kt_jvm_binary(
         "//scripts/src/java/org/oppia/android/scripts/docs:kdoc_validity_check_lib",
     ],
 )
+
+TODO_EXEMPTION_ASSETS = generate_todo_assets_list_from_text_protos(
+    name = "todo_exemption_assets",
+    todo_exemptions_name = ["todo_exemptions"],
+)
+
+kt_jvm_binary(
+    name = "todo_check",
```

Comment:
Here & everywhere else (including comments, protos, and file names): 'todo check' is a bit too generic, I think. I suggest instead: 'todo_open_check' since that's actually what's being done, here. Technically it's also a validity check, but that's not the main purpose.

Reply:
Done.